src/custom/hooks/useRefetchPriceCallback.ts

```diff
@@ -67,12 +90,31 @@ export function useRefetchQuoteCallback() {
           fee
         })
       } catch (error) {
-        console.error('Error getting the quote (price/fee)', error)
+        const errorType = error?.errorType
+        console.error('Error getting the quote (price/fee):', errorType)
+
+        // Unsupported token
+        if (errorType === ApiErrorCodes.UnsupportedToken) {
+          // TODO: will change with introduction of data prop in error responses
+          const unsupportedTokenAddress = error?.description?.split(' ')[2]
```

Comment:
Can you do a regex match against the token hex pattern instead?
I know this is temporary but feels soooo error prone :see_no_evil:

Reply:
heheheh. tired coding, ill do it tomorrow

Reply:
But this info will come in the data. I agree with the comment 100%, but I would be OK waiting until the backend has this info directly